neuralprophet/forecaster.py

```diff
@@ -3045,21 +3032,22 @@ def conformalize(self, df_cal, alpha, method="naive", plotting_backend="default"
                     * ``matplotlib``: Use matplotlib backend for plotting
                     * ``default`` (default): Use matplotlib backend for plotting
         """
-        df_cal = self.predict(df_cal)
+        # conformalize
+        df_cal = self.predict(calibration_df)
         if isinstance(plotting_backend, str) and plotting_backend == "default":
             plotting_backend = "matplotlib"
-        if self.config_conformal is None:
-            self.config_conformal = configure.Conformal(
-                method=method,
-                q_hats=conformalize(df_cal, alpha, method, self.config_train.quantiles, plotting_backend),
-            )
-        else:
-            self.config_conformal.method = method
-            self.config_conformal.q_hats = conformalize(
-                df_cal, alpha, method, self.config_train.quantiles, plotting_backend
-            )
-
-    # def conformalize_predict(self, df, df_cal, alpha, method="naive"):
-    #     self.conformalize(df_cal, alpha, method)
-    #     df_forecast = self.predict(df)
-    #     return df_forecast
+        q_hats = conformalize(df_cal, alpha, method, self.config_train.quantiles, plotting_backend)
+        # predict
+        df = self.predict(df)
```

Comment:
we should allow users to pass arguments to the prediction function, also see comment above.

Reply:
How about pass in **kwargs into `conformal_predict()`, which will then be passed in the `self.predict()`?

Reply:
Implemented.